src/SosThreadingTools/DumpAsyncCommand.cs

```diff
@@ -289,50 +298,53 @@ private static void ChainStateMachinesBasedOnJointableTasks(DebuggerContext cont
                 catch (Exception ex)
 #pragma warning restore CA1031 // Do not catch general exception types
                 {
-                    context.Output.WriteLine($"Fail to fix continuation of state {stateMachine.StateMachine.Address:x} Error: {ex.Message}");
+                    Console.WriteLine($"Fail to fix continuation of state {stateMachine.StateMachine.Address:x} Error: {ex.Message}");
                 }
             }
         }
     }
 
-    private static void MarkThreadingBlockTasks(DebuggerContext context, List<AsyncStateMachine> allStateMachines)
+    private void MarkThreadingBlockTasks(List<AsyncStateMachine> allStateMachines)
     {
-        foreach (ClrThread? thread in context.Runtime.Threads)
+        foreach (ClrRuntime runtime in this.Runtimes)
```

Comment:
The diff got confused below, I'm not sure why.  The only real change to this function was removing `context` and adding this `foreach`.